generators/app/templates/base/.github/workflows/pull-req-check.yml

```diff
@@ -41,29 +54,33 @@ jobs:
             ${{ runner.os }}-
 
 
-      # Install Dependencies
       - name: Install dependencies
         if: steps.cache-node-modules.outputs.cache-hit != 'true'
         run:  npm install
+      
 
-      # Run lint test
-      - name: Run lint Test if present only
+      - name: create ENV file
+        run: |
+          cat >> .env <<EOF
+          ${{ secrets.P2_<<service-name>>_ENV }}
+          EOF
+  
+  
+      - name: Run lint test if present only
         run:  npm run lint --if-present
 
-      # Run unit test
+    
       - name: Run unit test if present only
         run:  npm run test --if-present
 
-<% if (useSonarCloud) { %>
```

Comment:
we still need this here in the template
it is used to not include the sonarcloud check if the user doesn't choose sonarcloud